SnapClean3.py

```diff
@@ -266,6 +277,8 @@ def execute(self):
         # capture completion time
         finishTime = datetime.now().replace(microsecond=0)
 
+        self.logger.info('Total Snapshots in use with AMIs : %s', in_use_snapshots_count)
```

Comment:
1) I was confused by the name 'in_use_snapshot_count'.  Would you mind changing the name to 'snapshot_count_associated_with_AMIs' ?
2) like all of the other results, would you mind stashing  napshot_count_associated_with_AMIs in the results list, like we are doing for all of the other results we report on?  That will keep things consistent.
3) would you mind putting these two new totals after the existing 3 totals below?

Reply:
1) Am no longer using *_count variables in the script.
results[TOTAL_SNAPSHOTS_ASSOCIATED_WITH_AMIS] and 
results[TOTAL_INSCOPE_SNAPSHOTS_ASSOCIATED_WITH_AMIS] are the count values.

2) Done.

3) Done.

[2019-06-25 04:01:14,854][INFO]Total Snapshots inspected 267
[2019-06-25 04:01:14,854][INFO]Expired Snapshots 133
[2019-06-25 04:01:14,854][INFO]Deleted Snapshots 0
**[2019-06-25 04:01:14,854][INFO]Total Snapshots in use with AMIs : 1144
[2019-06-25 04:01:14,854][INFO]Total In-scope Snapshots currently in use with an AMI : 3**
[2019-06-25 04:01:14,854][INFO]Exceptions Encountered 0

Let me know if this looks good or needs some more work?